### PR TITLE
[fastlane] FastlaneCore::Simulator reset by version 2/2

### PR DESCRIPTION
**Pull request by hjanuschka**
# PR 2/2
Split of: https://github.com/fastlane/fastlane/pull/7194

as requested here: #7192

```ruby
lane :sim do
        reset_simulators(
                ios: ["8.1", "10.1"]
        )
        reset_simulators
end
```

  * 1/2 -> https://github.com/fastlane/fastlane/pull/7196
  * 2/2 -> https://github.com/fastlane/fastlane/pull/7197

### CODE REVIEW

**Comment by mgrebenets** [comment]
👍

File: fastlane/lib/fastlane/actions/reset_simulators.rb
Comment (milch) [inline fastlane/lib/fastlane/actions/reset_simulators.rb]: We should make it clear here that this corresponds to the `Reset content and settings...` option in the simulator menu bar, and is different from the `snapshot reset_simulators --ios ...` option.

**Review by milch — CHANGES_REQUESTED** [review]
Looks good, but let's merge the alias PR first and then rename this action + add the alias in one go. Just so that no one gets the wrong idea if they just have this change.

**Comment by hjanuschka** [comment]
@milch feedback adressed, action renamed and alias added  - do not merge before #7217  is merged.

**Review by milch — APPROVED** [review]

**Comment by milch** [comment]
@hjanuschka Great, thanks! 🚀

File: fastlane/lib/fastlane/actions/reset_simulator_contents.rb
Comment (milch) [inline fastlane/lib/fastlane/actions/reset_simulator_contents.rb]: `rubocop` demands empty lines between method definitions here

File: fastlane/lib/fastlane/actions/reset_simulator_contents.rb
Comment (milch) [inline fastlane/lib/fastlane/actions/reset_simulator_contents.rb]: Same here

**Review by milch — CHANGES_REQUESTED** [review]
Let's please the ruby style gods

**Comment by hjanuschka** [comment]
@milch updated, this was the first file for a long time i commited without rubocop'ing it first - and straight to a fail 😫

**Comment by milch** [comment]
You can put the following file into your `.git/hooks` folder as `pre-commit` if you want:

```bash
#!/bin/bash

bundle exec rubocop
bundle exec rake test_all
```

**Comment by hjanuschka** [comment]
@milch  thx, but i try to avoid git hooks - as they hardly sync across machines.

can you approve the changes the alias PR got merged 👍

**Review by milch — APPROVED** [review]

**Pull request merged**
